src/ios/MultipeerConnectivity/THEMultipeerServerSession.h

```diff
@@ -0,0 +1,41 @@
+//
+//  The MIT License (MIT)
+//
+//  Copyright (c) 2015 Microsoft
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+//  Thali CordovaPlugin
+//  THEMultipeerServerSession.h
+//
+
+#import "THEMultipeerPeerSession.h"
+
+// The sesssion type managed by a server (and so references a client)
+@interface THEMultipeerServerSession : THEMultipeerPeerSession
+
+// The local port on which the application server is listening
+@property (nonatomic) uint serverPort;
```

Comment:
NON-BLOCKING - Just for my own edification, why make this nonatomic? I recognize that it should only be set one time so atomic isn't really necessary. But is it just purely a perf thing?